scripts/change_file_ext_test.py

```diff
@@ -0,0 +1,164 @@
+# coding: utf-8
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Unit tests for scripts/change_file_ext.py."""
+
+import os
+import shutil
+
+import change_file_ext # pylint: disable=relative-import
+from core.tests import test_utils
+
+
+class ChangeExtensionsTest(test_utils.GenericTestBase):
+    """Test the change extension method."""
+
+    def setUp(self):
+        """Create dummy folders for testing and run the change extensions method
+        on the dummy files.
+        """
+        super(ChangeExtensionsTest, self).setUp()
+
+        self.rename_dir = 'dir_used_in_renaming'
+        self.non_rename_dir = 'dir_not_used_in_renaming'
+
+        os.mkdir(self.rename_dir)
```

Comment:
I see that you are making up dirs and folders. I am not sure about this (though this might not be that big a concern). But have you tried the `tempfile` module of python? It makes up temporary dirs and files which can be then safely deleted. It is used in [_pylint_extensions_test.py_](https://github.com/oppia/oppia/blob/develop/scripts/pylint_extensions_test.py). 
Thanks!

Reply:
I think it is better to use `tempfile` module. I have made the changes, PTAL.